test/NuGet.Core.Tests/NuGet.Configuration.Test/SettingsUtilityTests.cs

```diff
@@ -254,6 +254,33 @@ public void DeleteConfigValue_WithNullSettings_Throws()
             ex.Should().BeOfType<ArgumentNullException>();
         }
 
+        [Fact]
+        public void DeleteConfigValue_WithValidSettings_DeletesKey()
+        {
+            // Arrange
+            var keyName = "dependencyVersion";
+            var nugetConfigPath = "NuGet.Config";
+            var config = @"<?xml version=""1.0"" encoding=""utf-8""?>
+<configuration>
+  <config>
+    <add key=""" + keyName + @""" value=""Highest"" />
+  </config>
+</configuration>";
```

Comment:
var config = $@"
<configuration>
    <config>
        <add key=""{keyName}"" value=""Highest"" />
    </config>
</configuration>";

Reply:
I actually agree that this is easier to read, but in recent PR reviews, I've learned it's worse performance. Since this is a test, perf isn't really critical here, and it's probably negligible anyway, but I don't see enough value to rerun CI since it's already green.

Reply:
Unless the compiler has improvements I don't know about, interpolated strings (strings starting with `$`) are syntactic sugar for `string.Format`, which has much slower runtime performance than `string.Concat` (which `operator +(string a, string b)` is syntactic sugar for).

This is a test, so it's not a perf-critical hot-path. But, unless you find concatenated strings much less readable than interpolated strings, I think it's fine as-is.